### PR TITLE
Use post for long solr requests. Fixes #410

### LINKED ISSUE #410: Solr requests are too long on the work show page
```
ActionView::Template::Error (RSolr::Error::Http - 400 Bad Request
Error:

URI: http://localhost:8080/hydra/select?wt=ruby&rows=1000&q=%7B%21terms+f%3Did%7Dhh63sw059%2Ct148fh40q%2C73666459f%2C2n49t1770%2Cms35t878w%2Cgf06g278x%2Cdj52w483c%2C3b5918681%2Ccf95jb541%2Cth83kz498%2Cpv63g035z%2Cnz805z77p%2C5x21tf49n%2Cxg94hp70s%2Cqr46r0938%2Cb2773v80s%2Cvx021f20b%2Crv042t18s%2Cst74cq62z%2Ck0698758p%2Cvm40xr673%2Ckp78gg52v%2C2n49t1788%2C5q47rn90w%2Cv405s9450%2Cv118rd72n%2Ck0698759z%2Cs4655g667%2Cpr76f353n%2Cwp988j973%2C4x51hj16f%2C02870v95n%2C7w62f835x%2Ch702q646r%2Cft848q740%2Cg445cd27n%2Cbr86b374n%2Cx920fw93c%2Cdj52w484n%2Cms35t8795%2C8910jt673%2Cd504rk45z%2C8049g5214%2C37720c86f%2Cn296wz34n%2Cqn59q415v%2Czc77sq17w%2C4x51hj17q%2C5h73pw14h%2C2v23vt48v%2C0g354f345%2Cgm80hv476%2C0k225b16g%2Cjh343s452%2Cgm80hv48g%2Ck930bx145%2Ch989r338j%2C2514nk62v%2C5q47rn915%2C8k71nh22q%2Cd217qp60t%2Czk51vg86k%2Cv979v3282%2Cqj72p722r%2Cxd07gs84x%2C5712m667r%2C5138jd988%2Chh63sw06k%2Ckd17ct01m%2Cm900nt60x%2Cht24wj56k%2C9c67wm919%2C1j92g764r%2Cdv13zt31f%2Cm326m1871%2C2r36tx640%2C4t64gn319%2Cq237hs04k%2Cqn59q4164%2Cjs956f926%2Cw37636924%2C9s161630z%2Crf55z776g%2Cgx41mh976%2Ck3569441r%2C9c67wm92k%2Cxw42n802w%2C3j333238s%2Cn870zq95n%2Czc77sq185%2C8c97kq54h%2Ctt44pm955%2Crv042t192%2Ck0698760q%2Cmw22v5563%2Cf1881m08j%2C5x21tf50d%2Cmc87pq38w%2Cgf06g2796%2C79407x34w%2C2n49t179j%2C8336h2050%2C1z40ks886%2Ct148fh410%2C5712m6681%2Cpz50gw28m%2Cbv73c049j%2Cst74cq637%2C5d86p036g%2C5h73pw15s%2Chx11xf37m%2C2j62s501b%2Cjs956f93g%2C2n49t1809%2Ch989r339t%2C08612n65d%2C9g54xh819%2Cn296wz35x%2Cmw22v557c%2Ckp78gg534%2C1c18df87m%2Cbg257f135%2Cv405s9468%2C4q77fr52t%2Cbn9996823%2Cqf85nb47c%2Cng451h65t%2C47429937c%2Cfj2362271%2Cwh246s23s%2Ctq57nr113%2Cgq67jr31t%2Cqn59q417d%2Ctt44pm96f%2C4q77fr533%2C8s45q889n%2C1n79h437c%2Cdz010q170%2Cn583xv088%2C3484zg98v%2Csq87bt77b%2Cdf65v797g%2Ct722h8974%2Cnc580m74j%2Cdr26xx50d%2Csf268521w%2C3484zg994%2Chm50tr86j%2Cpc289j157%2Cgm80hv49r%2C1544bp19x%2Ck35694421%2Cb5644r666%2C2f75r816q%2Czg64tm03s%2Cfj2362289%2Cp8418n33n%2Chq37vn67v%2Cd791sg323%2Chm50tr87t%2C2801pg46q%2Crf55z777r%2Cns0646132%2C1j92g7651%2Ctb09j580g%2Cht24wj57v%2Cfx719m53g%2C5m60qr979%2C4x51hj180%2Cjw827b762%2C5t34sj71x%2Cp2676v61b%2C1v53jx12t%2C1c18df88w%2Cst74cq64h%2Cx633f110r%2Czw12z5417%2Cww72bb60x%2Cjh343s46b%2Cg158bh396%2C9k41zd566%2C0c483j52v%2C4t64gn32k%2Cr494vk24x%2C0k225b17r%2C5h73pw162%2Ctm70mv27g%2Cn296wz366%2C2227mp83c%2C1g05fb76b%2C5t34sj726%2Ckh04dp79j%2C7d278t156%2Czp38wc731%2C5712m6699%2Cf1881m09t%2Cdv13zt32q%2Cst74cq65s%2C41687h57c%2Cq237hs05v%2Cgt54kn14d%2Cqb98mf60p%2Cc821gj87v%2Cs4655g67h%2C8g84mm48m%2Cpk02c991b%2Cpk02c992m%2Crb68xb95f%2Ccc08hf774%2C1n79h438n%2Cz890rt448%2C1r66j1263%2C4b29b611r%2Chh63sw07v%2Czs25x8542%2C4b29b6121%2Cv979v329b%2Ckw52j8202%2C4f16c2963%2Cgt54kn15p%2C37720c87q%2Ckk91fk66p%2Csq87bt78m%2Cmw22v558n%2C9g54xh82k%2Crn301146c%2C4m90dv57p%2C7p88cg66f%2Cwm117p04t%2Cw6634379t%2Cmk61rh10p%2Cc247ds22r%2Cgt54kn16z%2C7m01bk84w%2Cks65hc42v%2Cjq085k08p%2Cpg15bd999%2Cvt150j44p%2C8336h2068%2Czw12z542h%2C9z902z99z%2C9019s252s%2Cg158bh40z%2Cv118rd73x%2C5138jd99j%2C79407x355%2C6m311p405%2Cr494vk256%2C9z903000p%2Cv692t635z%2Chm50tr883%2C0r967387z%2Crb68xb96q%2Cgq67jr323%2Cgh93gz59r%2C9w0323196%2Cmg74qm227%2Cft848q758%2Cd217qp613%2Cht24wj584%2C8g84mm49w%2C1z40ks89g%2C6d56zw72t%2Ccn69m426m%2Cqz20ss644%2C6h440s548%2Chh63sw084%2Cjs956f94r%2Czc77sq19f%2Csx61dm42q%2Cqn59q418p%2C8k71nh230%2Ctt44pm97q%2Cs1784k86g%2Cc247ds231%2C4b29b6139%2C9880vr05x%2Cc821gj884%2Ccz30ps84v%2Cj098zb20z%2C70795776v%2Cf1881m10k%2Ccc08hf78d%2C3197xm18w%2Cqz20ss65d%2Cvd66w0025%2C6q182k26v%2C707957774%2C8k71nh248%2C3484zh00p%2Ccf95jb559%2Csf2685225%2Cfn106z10x%2Ch415p9660%2C2514nk634%2Cx633f1111%2C02870v96x%2Cp8418n34x%2C9306sz356%2Cvt150j45z%2Cf1881m11v%2Cwp988j98c%2C4q77fr54c%2Cft848q76j%2C4x51hj198%2C5h73pw17b%2C8s45q890d%2Cv692t6367%2Ck35694439%2Cpn89d6699%2Cgh93gz60h%2C9k41zd57g%2C1831ck13h%2Cq811kj78n%2C8049g522d%2Csx61dm430%2C8623hx837%2Cv118rd746%2C5138jf003%2C6w924b947%2Cx633f1129%2C2f75r8170%2C2f75r8188%2C8p58pd05k%2Csb3978368%2Cpv63g0367%2Cn583xv09j%2C2v23vt494%2Chh63sw09d%2C4b29b614k%2Cpv63g037h%2Cq811kj79x%2C9019s2532%2Czp38wc749%2Cz029p4893%2C4q77fr55n%2C9w032320z%2Cfn106z116%2Cfb4948535%2Csb397837j%2C9p290947w%2Ckp78gg54d%2C2b88qc29j%2Ck930bx15f%2Chm50tr89c%2C1z40ks907%2C47429938n%2Cmk61rh11z%2C2801pg470%2C6969z0952%2C6682x406v%2C79407x36f%2C7w62f8366%2Cqb98mf61z%2Cdj52w485x%2Cqr46r094j%2Cxd07gs856%2Cx633f113k%2Cxd07gs86g%2C79407x37q%2Cjw827b77b%2Ckp78gg55p%2Cf4752g86z%2C2227mp84n%2C1j92g7669%2Cqf85nb48n%2Cwp988j99n%2Cgq67jr33c%2Cg445cd28x%2Cb2773v812%2C12579s352%2Cdz010q188%2Cgt54kn177%2C4t64gn33v%2Cw95050546%2Cj38607082%2Cdz010q19j%2C3b5918699%2Crn301147n%2Cb5644r67g%2Cgh93gz61s%2Cg732d910q%2Czc77sq206%2Ck643b130g%2Ctm70mv28r%2C8s45q891p%2C1831ck14s%2Crf55z7781%2Cv405s947j%2Cb2773v82b%2C8p58pd06v%2Czp38wc75k%2C3b5918702%2C8c97kq55s%2Ccc08hf79p%2C5999n353w%2C2j62s502m%2C47429939x%2C6t053g15k%2Ctb09j581r%2C6h440s55j%2Cmg74qm23h%2C9880vr066%2Cbk128b02w%2C3n203z20d%2Czg64tm042%2Czp38wc76v%2Cz603qx49f%2Cgx41mh98g%2Cpn89d6702%2Cqb98mf627%2C3f462548s%2C4x51hj201%2Crn301148x%2Cdn39x1667%2Cdf65v798r%2Cpc289j16h%2C2r36tx658%2Csn009x90d%2Cpr76f354x%2Cvm40xr68c%2Cr781wg10t%2Cff365539k%2Chq37vn684%2Cht24wj59d%2Cnc580m75t%2C4b29b615v%2C4x51hj219%2Cn296wz37g%2C2801pg488%2C0k225b181%2Ccn69m427w%2C2227mp85x%2Cp5547r609%2Cs7526c64x%2Cnz805z78z%2Cf1881m124%2Cz316q170m%2Ctq57nr12c%2Cn583xv109%2Cw6634380k%2Cws859f86t%2C1c18df895%2Crb68xb970%2Cf7623c688%2C5t34sj73g%2C6h440s56t%2Cs7526c656%2C0k225b199%2Cg445cd296%2Cjs956f951%2C05741r79h%2C12579s36b%2C6108vb34k%2Cv979v3303%2Ck06987610%2C7h149q04x%2Crr171x33n%2C9p2909485%2Ch702q6471%2Crn3011496%2Cqn59q419z%2C47429940p%2Cbr86b375x%2Csb397838t%2Cqj72p7231%2C6h440s573%2Cpz50gw29w%2C9593tv20s%2C3x816m69m%2C02870v976%2Ccn69m4285%2Cwp988k006%2Ccr56n1117%2Csf268523f%2Ctb09j5821%2Cwm117p053%2Cv979v331c%2Cj96020713%2C5m60qr98k%2Cmp48sc852%2Ccz30ps854%2Cb5644r68r%2Cdr26xx51p%2C9k41zd58r%2Cv979v332n%2Ch415p9678%2Crx913q00j%2Cc534fp131%2Cns064614b%2Cz316q171w%2C3x816m70c%2Cr494vk26g%2Cd791sg33c%2Csq87bt79w%2Cp5547r61k%2Cp8418n356%2Cr781wg113%2Chq37vn69d%2C9019s254b%2Cxk81jk477%2Cns064615m%2C1g05fb77m%2Ctb09j5839%2C4q77fr56x%2C7d278t16g%2Cxw42n8035%2Cnc580m763%2C2z10wq326%2C7m01bk855%2C6108vb35v%2Cs7526c66g%2C3n203z21p%2Cnp193930g%2C1v53jx133%2Cbk128b035%2Csf268524q%2Cf4752g877%2C8910jt68c%2C4b29b6164%2C9k41zd591%2C1j92g767k%2Cqb98mf63h%2C3b591871b%2Cb8515n477%2C9p290949f%2Cz603qx506%2Cdz010q209%2C7m01bk86f%2Cz029p490v%2Cr207tp446%2Cr494vk27r%2Cmk61rh127%2Cm326m1889%2C8910jt69n%2Crr171x34x%2Ckp78gg56z%2Cmc87pq395%2C37720c880%2Cg158bh417%2C8910jt70d%2Cbv73c0509%2Cj6731393w%2Ct435gd107%2Crr171x356%2Cpc289j17s%2Cd504rk467%2C79407x380%2Crn301150z%2Cjw827b78m%2C1v53jx14c%2Cmg74qm24s%2Cd217qp62c%2Cvq27zn59x%2Czk51vg87v%2Cws859f873%2C3f4625492%2Cs7526c67r%2Cjm214p263%2C2j62s503w%2C2r36tx66j%2Ch415p968j%2C6h440s58c%2C3j3332392%2Cks65hc434%2C1c18df90x%2Cdv13zt330%2Czg64tm05b%2Cpg15bf00v%2Cgq67jr34n%2Cth83kz501%2Cmw22v559x%2Cnv935296t%2Cks65hc44d%2C9306sz36g%2C9880vr07g%2Cwp988k01g%2C7p88cg67q%2C5d86p037r%2Cxs55mc14f%2C12579s37m%2Crj4304685%2C2z10wq33g%2C1n79h439x%2Cwh246s242%2C05741r808%2Cgb19f5917%2C7p88cg680%2C70795778d%2Chh63sw105%2C8w32r571f%2C8910jt71p%2Cs1784k87r%2Cvx021f21m%2Cmp48sc86b%2Ctb09j584k%2Ctq57nr13n%2Cmg74qm252%2Cp8418n36g%2C9z903001z%2C8w32r572q%2Cb5644r691%2C1831ck152%2Cwd375w37w%2C47429941z%2Ccr56n112h%2Cvq27zn60p%2Cxg94hp712%2Cgt54kn18h%2C7m01bk87q%2Cjw827b80n%2C0p096705g%2Cdn39x167h%2Cq811kj80p%2C05741r81j%2Ch128nd84p%2Cwp988k02r%2C0r967389h%2Cbv73c051k%2Cd791sg34n%2C3r074v020%2C9306sz37r%2Cg732d9110%2Cnc580m77c%2C00000012s%2Chx11xf38w%2Cn296wz38r%2Cbz60cw39c%2C1544bp20p%2C8g84mm50n%2Cqv33rw73v%2Cd217qp63n%2C9k41zd60s%2Cft848q77t%2Cbr86b3766%2Crn3011517%2Cd791sg35x%2C8s45q892z%2Czg64tm06m%2Cxg94hp72b%2Cwp988k031%2Crr171x36g%2Crv042t20t%2C6682x4074%2Cmk61rh13h%2Cr781wg12c%2Cst74cq662%2C5712m6702%2Cxd07gs87r%2C0c483j54d%2Cjq085k09z%2C474299427%2C7w62f837g%2Cdf65v7991%2C6w924b95h%2C5d86p0399%2C3197xm195%2C765371485%2C9p2909506%2Chh63sw11f%2C9g54xh83v%2Chx11xf395%2Cm900nt616%2Cb8515n48h%2Cfj236230b%2Ctx31qh84w%2Csn009x91p%2Cn009w243k%2C5425k977r%2Cfn106z13r%2Cqb98mf64s%2Cqf85nb49x%2Cq524jn85t%2C1r66j127c%2Czp38wc774%2Chm50tr904%2Cgh93gz63b%2Cdb78tc05q%2C1g05fb795%2C5q47rn92f%2Csf2685250%2Ch415p969t%2C8w32r5730%2C1g05fb80x%2Cs7526c681%2C1r66j128n%2Cqf85nb50p%2Cv979v333x%2Cj098zb217%2C6h440s60d%2C6h440s61p%2C6108vb364%2Cbc386j28j%2Cq237hs064%2Cpz50gw30n%2C7h149q056%2Cwm117p06c%2Cs1784k881%2Cm326m190b%2Cvh53wv84h%2C08612n66p%2Crb68xb988%2C3r074v038%2C2227mp87g%2Cgf06g280z%2Cd217qp64x%2Ck06987628%2Cxd07gs881%2Cnc580m79x%2Chd76s016r%2Ccz30ps86d%2C8049g523p%2Cgh93gz64m%2Cw6634383d%2Cpv63g038s%2Chq37vn705%2Cpg15bf014%2Ch415p970k%2Cv979v3346%2Cxk81jk48h%2Cwp988k049%2Ctb09j585v%2Cp5547r62v%2Cs4655g68s%2Cqv33rw744%2Cxw42n804f%2Czp38wc78d%2Chm50tr91d%2Cw0892b079%2Cg158bh42h%2Cns064616w%2Ch702q6489%2C6682x408d%2C9593tv212%2Cfq977t96s%2C0v838072k%2C00000014b%2Cv405s948t%2Cq811kj81z%2Cn009w244v%2C0c483j55p%2C2z10wq34r%2Cwh246s25b%2Cqv33rw75d%2Cwm117p07n%2Cws859f88c%2Ch989r340k%2C2801pg49j%2Cc821gj89d%2Cj3860709b%2C7s75dc47g%2C08612n67z%2Ct148fh428%2Cc247ds249%2Cqr46r095t%2Cm326m191m%2Ctd96k261g%2C6395w719p%2C6395w720f%2Cwp988k05k%2Cmp48sc87m%2C5d86p0402%2C4m90dv597%2Cnk322d50f%2Ct148fh43j%2C7m01bk880%2C5m60qs00d%2Cj9602072c%2Cqn59q420q%2C5138jf01c%2Cnc580m80p%2C4m90dv600%2Cj098zb22h%2Cj67313945%2Cjh343s47m%2Ct148fh44t%2Crb68xb99j%2Cjd472w54s%2C6682x409p%2C2v23vt50w%2Cwh246s26m%2Cwd375w385%2Cbn999683c%2Czp38wc79p%2Cvq27zn61z%2Cxg94hp73m%2Cjh343s48w%2Cbv73c052v%2C1v53jx15n%2Cz890rt45j%2Ck3569444k%2C0c483j56z%2Cfb494855q%2C9306sz381%2Cxp68kg31v%2Cnp193931r%2Cs1784k899%2Cs1784k902%2Cw0892b08k%2Cgx41mh99r%2C5x21tf52z%2Cr494vk281%2Ckh04dp81k%2Cb2773v83m%2C70795779p%2C736664606%2Cv405s9493%2Cmp48sc88w%2C0c483j577%2C3197xm20x%2Cgq67jr35x%2C9019s255m%2C9c67wm93v%2Cg732d9128%2Cdv13zt348%2Chd76s0171%2C9p290951g%2Cb2773v84w%2Cnv9352973%2C5425k9781%2C9306sz399%2Ct722h898d%2Crj4304706%2C02870v98g%2Cbn999684n%2C9w0323217%2C6108vb37d%2Cc821gj905%2Cht24wj605%2C6969z097m%2Cx920fw94n%2Cm326m192w%2Ch128nd85z%2C5q47rn93q%2Crn301152h%2C6q182k274%2Cq811kj827%2C0p096706r%2Ct722h899p%2Cxp68kg324%2C4t64gn344%2Ckh04dp82v%2C9p290952r%2C4t64gn35d%2C0v838073v%2Ccn69m429f%2Cqz20ss66p%2C4b29b617d%2Cp2676v63w%2C2v23vt515%2C7s75dc48r%2C8k71nh26t%2Cst74cq67b%2Csn009x92z%2Cq524jn863%2Cn870zq96x%2Cx059c747x%2C0v8380744%2C6t053g16v%2C9p2909531%2Cjs956f969%2Cz890rt46t%2C2f75r819j%2Cgq67jr366%2C8c97kq562%2C3n203z22z%2C3197xm216%2Cz890rt473%2Cr494vk299%2C9593tv22b%2C41687h58n%2C1c18df92g%2C1g05fb82g%2Cb8515n49s%2C1r66j129x%2Cjm214p27c%2Cmk61rh14s%2Cq524jn87c%2Crn301153s%2C4x51hj22k%2C0z708w53f%2Ckp78gg577%2C1831ck16b%2Cbk128b04f%2C3n203z237%2C2v23vt52f%2Cn870zq976%2Cv118rd76r%2C8k71nh273%2Crj430471g%2C79407x398%2Cd217qp656%2C5q47rn940%2C9c67wm944%2Cwm117p08x%2C8g84mm51x%2Cj38607103%2Cp5547r634%2Cg732d913j%2Cgh93gz65w%2C1544bp21z%2Cvd66w003f%2Cm613mx716%2Csf2685268%2Cpz50gw31x%2C9306sz402%2Cpc289j182%2Cnc580m81z%2Ck930bx16q%2Cvx021f22w%2Cw9505055g%2C2514nk64d%2Cgt54kn19s%2Cpn89d671b%2C9019s256w%2Cmc87pq40x%2Czk51vg884%2C4j03cz787%2Cvq27zn627%2C6w924b96s%2C9z9030027%2C2n49t182v%2C76537149f%2Cvd66w004q%2Cmg74qm26b%2Cxk81jk49s%2Cth83kz52k%2Czg64tm085%2Cx059c7486%2Czp38wc80f%2C6m311p41f%2C8g84mm526%2Cp8418n37r%2Csx61dm448%2C0r9673908%2Cth83kz53v%2Cm613mx72g%2Cjh343s495%2C7w62f838r%2Cnc580m827%2C7m01bk898%2Cxs55mc15q%2C2v23vt53q%2Cdf65v800k%2C1c18df93r%2C3197xm22g%2C9306sz41b%2Cww72bb616%2Cq237hs07d%2C0p0967071%2Ctt44pm980%2C3b591872m%2C9z903003h%2Cwp988k06v%2C00000015m%2Ch415p971v%2C47429943h%2Chq37vn71f%2Cpr76f3556%2Cz890rt48c%2Cgt54kn20j%2Cws859f89n%2Cxd07gs899%2Cgq67jr37g%2C2f75r8209%2C2f75r821k%2Cnv935298c%2C2z10wq351%2C9k41zd612%2C2f75r822v%2C8910jt72z%2Cfj236231m%2Cbg257f14f%2Cg158bh43s%2Cb5644r70s%2Cp2676v645%2C8623hx84h%2Cfb4948560%2C02870v99r%2Cw0892b09v%2C5d86p041b%2Csb397840v%2Cx346d428z%2Cgq67jr38r%2C9p2909549%2Cs7526c699%2C0p0967089%2Ctx31qh855%2Cqf85nb51z%2C4b29b618p%2C1v53jx16x%2Cdb78tc060%2Cd217qp66g%2Cbr86b377g%2Ccr56n113s%2Crj430472r%2Cpr76f356g%2Csf268527j%2Csq87bt80n%2Cfj236232w%2Cg732d914t%2Cn009w2454%2Cfn106z141%2C2f75r8234%2Ckd17ct02w%2Cj9602073n%2Cjs956f97k%2Cbr86b378r%2Ctq57nr14x%2Cq524jn88n%2Cbv73c0534%2C7w62f8391%2Ccv43nw98z%2Cz603qx531%2Cjw827b81x%2Cp2676v65f%2Cwd375w39f%2C9593tv23m%2Cd504rk47h%2Cht24wj61f%2Cbz60cw404%2C2j62s505f%2Cjh343s50x%2Cvm40xr69n%2Cw9505056r%2C1544bp227%2C9c67wm95d%2C4t64gn36p%2C3484zh027%2C3f462550t%2Ccv43nw997%2Cwh246s27w%2Cz316q1725%2Cnp1939321%2C6969z098w%2Crb68xc003%2Cf4752g88h%2Cpn89d672m%2C9s1616317%2Cm613mx73r%2Cnz805z800%2Ccf95jb56k%2Chx11xf40x%2C1z40ks91h%2Cxd07gs902%2Cc821gj91f%2C0p096709k%2C9p290955k%2Cj6731395f%2Cdv13zt35j%2C9019s2575%2Cj3860711c%2C3n203z24h%2C2b88qc309%2C6395w721q%2Cnc580m83h%2C5m60qs01p%2Cvx021f235%2Cjm214p28n%2Crb68xc01c%2C4q77fr576%2C4t64gn37z%2C9880vr091%2Cdf65v801v%2Cfx719m54r%2C2b88qc31k%2Cqz20ss67z%2C8w32r5748%2Ctt44pm998%2Cjd472w56b%2Cd791sg366%2Cxp68kg33d%2Cx920fw95x%2C4x51hj23v%2C9g54xh844%2Cnc580m84s%2Cjw827b826%2Cd791sg37g%2Cks65hc46z%2C0g354f35f%2Ch989r341v%2C9c67wm96p%2Ccc08hf80f%2C2n49t1834%2Cm039k5047%2Ckk91fk67z%2Cgm80hv50h%2Cn870zq98g%2C5h73pw18m%2Cp8418n381%2C8c97kq57b%2Csj139209v%2C3r074v04j&qt=standard

Backtrace: /opt/goldenseal/shared/bundle/ruby/2.2.0/gems/rsolr-1.0.13/lib/rsolr/client.rb:284:in `adapt_response'
/opt/goldenseal/shared/bundle/ruby/2.2.0/gems/rsolr-1.0.13/lib/rsolr/client.rb:190:in `execute'
/opt/goldenseal/shared/bundle/ruby/2.2.0/gems/rsolr-1.0.13/lib/rsolr/client.rb:176:in `send_and_receive'
(eval):2:in `get'
/opt/goldenseal/shared/bundle/ruby/2.2.0/gems/active-fedora-9.5.0/lib/active_fedora/solr_service.rb:107:in `query'
/opt/goldenseal/shared/bundle/ruby/2.2.0/bundler/gems/curation_concerns-bd710980eafb/app/presenters/curation_concerns/presenter_factory.rb:33:in `load_docs'
/opt/goldenseal/shared/bundle/ruby/2.2.0/bundler/gems/curation_concerns-bd710980eafb/app/presenters/curation_concerns/presenter_factory.rb:22:in `build'
/opt/goldenseal/shared/bundle/ruby/2.2.0/bundler/gems/curation_concerns-bd710980eafb/app/presenters/curation_concerns/presenter_factory.rb:8:in `build_presenters'
/opt/goldenseal/shared/bundle/ruby/2.2.0/bundler/gems/curation_concerns-bd710980eafb/app/presenters/curation_concerns/work_show_presenter.rb:29:in `file_presenters'
/opt/goldenseal/shared/bundle/ruby/2.2.0/bundler/gems/curation_concerns-bd710980eafb/app/views/curation_concerns/base/_related_files.html.erb:1:in `__opt_goldenseal_shared_bundle_ruby_______bundler_gems_curation_concerns_bd______eafb_app_views_curation_concerns_base__related_files_html_erb__1358295290537811671_54258000'
```

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: app/presenters/curation_concerns/presenter_factory.rb [hunk 1]
```diff
@@ -30,7 +30,16 @@ def build
...
       # @return [Array<SolrDocument>] a list of solr documents in no particular order
       def load_docs
-        ActiveFedora::SolrService.query("{!terms f=id}#{ids.join(',')}", rows: 1000).map { |res| SolrDocument.new(res) }
+        query("{!terms f=id}#{ids.join(',')}", rows: 1000)
+          .map { |res| SolrDocument.new(res) }
+      end
+
+      # Query solr using POST so that the query doesn't get too large for a URI
+      def query(query, args = {})
+        args.merge!(q: query, qt: 'standard', rows: 1000)
```
Comment (mjgiarlo) [inline app/presenters/curation_concerns/presenter_factory.rb]: Do you want `rows: 1000` on both lines 33 and 39?

**Comment by mjgiarlo** [comment]
One question: https://github.com/projecthydra-labs/curation_concerns/pull/411#discussion_r42540871

Otherwise :+1:

File: app/presenters/curation_concerns/presenter_factory.rb (hunk 1 above)
Comment (jcoyne) [inline app/presenters/curation_concerns/presenter_factory.rb]: I think just on row 33 will be fine. Good catch.

Comment (mjgiarlo) [inline app/presenters/curation_concerns/presenter_factory.rb]: Visually I kinda prefer it on line 39. Line 33 is already a bit eye-popping. But meh, either is fine.

Comment (jcoyne) [inline app/presenters/curation_concerns/presenter_factory.rb]: I'm trying to keep parity with ActiveFedora::SolrService#query.  I'd like to refactor that to optionally use POST, which allows us to remove the `query` method from this class.

**Pull request merged**